Index: src/com/classehomework/Time.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/classehomework/Time.java	(revision 6f34fc2547f01fba6e8731b3fc0ad1e33a63671d)
+++ src/com/classehomework/Time.java	(date 1559680602000)
@@ -61,11 +61,11 @@
                 ", second=" + second +
                 '}';
     }
-
-    public Time nextSecond() {
-        this.second += 1;
-        return;
-    }
+//
+//    public Time nextSecond() {
+//        this.second += 1;
+//        return;
+//    }
 
 
 }
